configuration/turn-on-auditing.md

```diff
@@ -28,6 +28,7 @@ After you complete the steps described, wait for audit logs to show on the Audit
 
 {% hint style="warning" %}
 **Please note!**  
+When you initially connect your tenant to Syskit Point, **Syskit Point collects audit logs no more than 7 days in the past**.
```

Comment:
maybe with an additional note that this is not up to us, it's simply a Microsoft limitation

> When you initially connect your tenant to Syskit Point, **Syskit Point collects audit logs no more than 7 days in the past. This is a limitation of the Microsoft Management Activity API.